### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by Sherpa67** [DineshSolanki/FoliCon#198] [Q&A]
Can any one help on how to uninstall folicon, as i need to move the folder i have got it saved to to another drive and then reinstall it on that drive....Thank you in advance for any help/advice

**Comment by DineshSolanki** [comment]
@Sherpa67 FoliCon is a portable application, meaning it doesn't require traditional installation or uninstallation. You can move the FoliCon folder to any drive without issues.
However, if you've added the FoliCon option to your context menu, follow these steps after moving the application:

Open FoliCon in its new location.
Select the "Remove context menu" option.
Then, select the "Add context menu" option again.

This ensures the context menu functionality works correctly with FoliCon's new location.
Regarding movie folders: You can freely move these to any location on your computer. The folders will function normally without requiring any additional changes.
